workflows/ci.yml

```
@@ -35,7 +35,7 @@ jobs:
       - name: Setup yq
         run: sudo snap install yq
       - name: Run pre-conditions
-        run: test -f .github/actions.yml && eval "$(yq r .github/actions.yml -D "true" pre.ci)" || true
+        run: test -f .github/actions.yml && eval "$(yq -r .pre.ci .github/actions.yml)" || true
```

Taking a look at the [`v3 to v4`](https://mikefarah.gitbook.io/yq/upgrading-from-v3#reading-with-default-value) docs, we should use and "evaluate" expression and recover the default value:

```suggestion
        run: test -f .github/actions.yml && eval "$(yq e '.pre.ci // "true"' .github/actions.yml)" || true
```

---

Fixed